api/pkg/s3/datastore/store.go

```diff
@@ -82,4 +82,6 @@ type DataStoreAdapter interface {
 	AbortMultipartUpload(multipartUpload *pb.MultipartUpload, context context.Context) S3Error
 
 	ListParts(listParts *pb.ListParts, context context.Context) (*model.ListPartsOutput, S3Error)
+	ListBackendObjects(context context.Context, prefix string, limit int64, marker string) (*pb.ListObjectResponse, S3Error)
+	GetBackendType() (string)
```

Comment:
No need to add `()` if only one type returned.

Reply:
It's removed.